crates/revm/src/evm.rs

```diff
@@ -195,10 +195,7 @@ impl<EXT, DB: Database> Evm<'_, EXT, DB> {
 
             let exec = &mut self.handler.execution;
             let frame_or_result = match next_action {
-                InterpreterAction::Call {
-                    inputs,
-                    return_memory_offset,
-                } => exec.call(&mut self.context, inputs, return_memory_offset),
+                InterpreterAction::Call { inputs } => exec.call(&mut self.context, inputs, 0..0),
```

Comment:
now that the `return_memory_offset` field is in `CallInputs`, the argument could be removed from `ExecutionHandler::call` right?

Reply:
> now that the `return_memory_offset` field is in `CallInputs`, the argument could be removed from `ExecutionHandler::call` right?

ty so much for your review & your time, does this looks better now?